src/web/lib/hakuneko/frontend@classic-dark/connectors.html

```diff
@@ -311,6 +311,8 @@
             _openLink(evt, link) {
                 if(link) {
                     let popup = window.open(link, evt.model.label, 'frame=true,nodeIntegration=no');
+                    // enable replaceAll only available in later chrome version
+                    popup.eval('String.prototype.replaceAll = function (search, replaceWith) { return this.split(search).join(replaceWith) }');
```

Comment:
This will not work for websites using `replaceAll(<RegExp>, <Functor>)` or any combination of the mentioned parameter types

Reply:
it's just a temp fix for viz, I propose to wait for haruneko for a full fix.